source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/projects/show/overview/products/item/index.tsx

```diff
@@ -57,6 +58,12 @@ class ProductItem extends React.Component<IProps> {
           >
             {name}
           </div>
+          {!isEmpty(publishLink) &&
+            <div
+            >
+              <a href={publishLink}>Link</a>
+             </div>
+          }
```

Comment:
@NullVoxPopuli: I am not sure how to style the link.  Suggestions on this section of HTML?  Also, I will turn "Link" into {t('Link')} or whatever you suggest as the text shown to user.

![image](https://user-images.githubusercontent.com/517814/51085109-8af20100-1702-11e9-8928-c2e4b418a1ec.png)

Reply:
@NullVoxPopuli Not sure how to add tests.

Reply:
we have some utility classes for spacing, I think `className=`m-l-md` for margin left medium should do it.
I think the text is fine, but I think I'd prefer this icon: https://material.io/tools/icons/?icon=launch&style=baseline "Launch", should be accessible from:

```ts
import LaunchIcon from '@material-ui/icons/Launch'
```
(or something close to that)

Reply:
@NullVoxPopuli: How is this?

![image](https://user-images.githubusercontent.com/517814/51093112-b958f600-176d-11e9-8712-10cce182611f.png)

Reply:
that looks nice!
iirc, the blue text for the product doesn't link to anything. since they are the same color, the blue text should probably also link to what the icon does

Reply:
I didn't intend for the icon to be blue.  Switching to use React components... learning...

Reply:
![image](https://user-images.githubusercontent.com/517814/51094160-aef02980-1778-11e9-8c2b-18918cd562af.png)

hover

![image](https://user-images.githubusercontent.com/517814/51094172-bf080900-1778-11e9-97bb-05169cd4ba4a.png)

Reply:
there is an existing test file at projects/show/__tests__/product-acceptance-test.ts, two additional tests asserting the presence / lack of presence of this link would be good. there is a page.ts in the __tests__ folder which has all the selectors and stuff for checking things in the dom. 

it _should_ be kinda straightforward, if not feel free to bug me in slack :)